frontend/pages/Homepage/Homepage.tsx

```diff
@@ -137,6 +137,24 @@ const Homepage = (): JSX.Element => {
     }
   );
 
+  const {
```

Comment:
This code using useQuery to get team enroll secrets appears in a couple other components in the codebase. Can we pull this out into a reusable hook? Im curious to see it pulling some of these useQuery calls in components can be a useful pattern for us.